scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/PPMLContext.scala

```diff
@@ -193,6 +193,46 @@ object PPMLContext{
     initPPMLContext(conf, appName)
   }
 
+  /**
+   * init ppml context with an existed SparkSession
+   * @param sparkSession a SparkSession
+   * @return a PPMLContext
+   */
+  def initPPMLContext(sparkSession: SparkSession): PPMLContext = {
+    val conf = sparkSession.sparkContext.getConf
+    val kmsType = conf.get("spark.bigdl.kms.type", defaultValue = "SimpleKeyManagementService")
+    val kms = kmsType match {
+      case KMS_CONVENTION.MODE_EHSM_KMS =>
+        val ip = conf.get("spark.bigdl.kms.ehs.ip", defaultValue = "0.0.0.0")
```

Comment:
Throw an error if this ip is not in conf. And also below port, key, id, etc.  This should be set in python side.